models/user.ts

```diff
@@ -10,8 +11,10 @@ interface UserAttributes {
   password: string
```

Comment:
User model should also have 'username' and 'email'

Reply:
Well it does have it, but for some reason it is not visible here. But it is visible here [https://github.com/miletic94/uteam-api/blob/021c5c0f5689ad9d1503a30c2573fa9dc8c4de81/models/user.ts](url)

Reply:
> User model should also have 'username' and 'email'

Well it does have it, but for some reason it is not visible here. But it is visible here https://github.com/miletic94/uteam-api/blob/021c5c0f5689ad9d1503a30c2573fa9dc8c4de81/models/user.ts